recipes/tcsh/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -eu -o pipefail
+
+ldconfig -v
+
+./configure 
```

Comment:
Need to add `--prefix="${PREFIX}"`. Otherwise it won't be looking in the right place for headers and libraries. :)